freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -545,7 +545,10 @@ def _get_csv_name(csv_name, version, rebuild_version, fm_suffix):
         :rtype: str
         """
         if version in csv_name:
-            return csv_name.replace(version, rebuild_version)
+            # The CSV name must be in the format of a valid DNS name, which means the + from the
+            # build ID must be replaced
+            dns_safe_rebuild_version = rebuild_version.replace("+", "-")
+            return csv_name.replace(version, dns_safe_rebuild_version)
         else:
             return f'{csv_name}.{fm_suffix}'
```

Comment:
would this also need changing?

Reply:
Nice catch! That's what happens when you rush a code change before a meeting. I learned my lesson.  :stuck_out_tongue:

Reply:
Thanks for pointing this out. I double checked and `fm_suffix` is the portion without the `+`.